com.editor.gamma.json

```diff
@@ -0,0 +1,56 @@
+{
+    "app-id": "com.editor.gamma",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "command": "/app/bin/gamma",
+    "finish-args": [
+    	"--share=network",
+    	"--socket=x11",
+		"--share=ipc",
+		"--socket=session-bus",
+		"--filesystem=host",
+		"--own-name=com.editor.gamma"
```

Comment:
```suggestion
```
You already own that name.

Reply:
I will change it to `com.github.hamadmarri.gamma`